pkg/kore/clusters.go

```
@@ -306,10 +309,16 @@ func (c *clustersImpl) ensureIdentifiers(ctx context.Context, cluster *clustersv
 		return false, nil
 	}
 
+	// work out the right cloud for the cluster provider so we can tag the asset correctly
+	cloud, mapErr := c.Costs().Metadata().MapProviderToCloud(cluster.Spec.Kind)
+	if mapErr != nil {
+		log.WithField("provider", cluster.Spec.Kind).WithError(mapErr).Warn("received error mapping k8s provider to cloud provider for asset identifier, will continue")
```

I don't think we should swallow this error. If we introduce a new cloud provider, this will help us to catch a missing mapping right away.

---

fixed in a3336ce